Externals.cfg

```diff
@@ -34,7 +34,7 @@ hash = 34723c2
 required = True
 
 [ccs_config]
-tag = ccs_config_cesm0.0.64
+tag = ccs_config_cesm0.0.65
```

Comment:
So is this correct? Or incorrect? Is it okay to update it?

Reply:
Yes, this is correct. And certainly fine to update it. I was just curious as to what the change was that you needed.

Reply:
Oh, I think it was a fix that @billsacks had suggested to use because one of the FATES runs was really slow. But perhaps this is not really needed anymore?

Reply:
Ahhh, I wonder if the slowdown needed the MPT update then. You should leave it in. We should do a tag with an externals update separate from this. But, you might as well leave this small change in hope that it does help.

Reply:
Okay thanks!